File: C++/Human.cpp
#include <iostream>
#include <string>

using namespace std;

class Human{
    
    private:
        string NIK;
        string name;
        string gender;

    public:
    // Constructor
        Human(){
            this->NIK = "";
            this->name = "";
            this->gender = "";
        }

        Human(string NIK, string name, string gender){
            this->NIK = NIK;
            this->name = name;
            this->gender = gender;
        }

    // Setter dan Getter

        // Untuk NIK
        string get_NIK(){
            return this->NIK;
        }

        void set_NIK(string NIK){
            this->NIK = NIK;
        }

        // Untuk Name
        string get_Name(){
            return this->name;
        }

        void set_Name(string name){
            this->name = name;
        }

        // Untuk Jenis Kelamin
        string get_Gender(){
            return this->gender;
        }

        void set_Gender(string gender){
            this->gender = gender;
        }

        // Destruktor
        ~Human(){
        }
    
};

File: Python/Human.py
class Human:
    
    # Inisialisasi variabel untuk setiap objek dari kelas
    NIK = ""
    name = ""
    gender = ""
    
    def __init__(self, NIK = "", name = "", gender = ""):
        self.NIK = NIK
        self.name = name
        self.gender = gender
        
    # Getter dan Setter untuk NIK
    def get_NIK(self):
        return self.NIK
    
    def set_NIK(self, NIK):
        self.NIK = NIK
        
    # Getter dan Setter untuk Name
    def get_Name(self):
        return self.name
    
    def set_Name(self, name):
        self.name = name
        
    # Getter dan Setter untuk Jenis Kelamin
    def get_Gender(self):
        return self.gender
    
    def set_Gender(self, gender):
        self.gender = gender